### PR TITLE
fix: clean up 

### PR DESCRIPTION
**Pull request by gabehamilton**
* clean up copy/share urls
 * summary component accepts action param (eliminates custom summary components)
 *  ipfs image support in EntitySummary

### CODE REVIEW

**Review by calebjacob — APPROVED** [review]

File: src/Entities/Template/EntityDetails.jsx [hunk 1]
```diff
@@ -186,6 +186,11 @@ const entityProperties = (obj) => {
 const listLink = href({
   widgetSrc: returnTo,
 });
+const detailsLink = href({
+  widgetSrc: `${REPL_ACCOUNT}/widget/Entities.Template.EntityDetails`,
+  params: { src, schemaFile, namespace, entityType, returnTo: homeLink },
+});
+const detailsUrl = "https://${REPL_NEAR_URL}/" + detailsLink;
```
Comment (shelegdmitriy) [inline src/Entities/Template/EntityDetails.jsx:193]:
```suggestion
const detailsUrl = `https://${REPL_NEAR_URL}/${detailsLink}`;
```

This suggestion will work the same but looks cleaner

File: src/Entities/Template/EntitySummary.jsx [hunk 2]
```diff
@@ -258,6 +261,28 @@ return (
             ),
           }}
         />{" "}
+        {entityActionUrlFunction && (
+          <Widget
+            src="${REPL_ACCOUNT}/widget/DIG.Tooltip"
+            props={{
+              content: "Use " + title,
```
Comment (shelegdmitriy) [inline src/Entities/Template/EntitySummary.jsx:268]:
```suggestion
              content: `Use ${title}`,
```

**Review by shelegdmitriy — APPROVED** [review]
LGTM! Added a few comments

File: src/Entities/Template/EntityDetails.jsx (hunk 1 above)
Comment (gabehamilton) [reply src/Entities/Template/EntityDetails.jsx:193]: I used concatenation here to highlight that the replacement of REPL_NEAR_URL is not via ES6 template. Also, it should be slightly faster, although we don't really care about performance here.

**Pull request merged**